### DISCUSSION TITLE
Latest update for custom client

### DISCUSSION THREAD

**Discussion by jasoncks** [mozilla/hubs#3803] [Hubs Cloud Questions] [answered]
We found that Hubs Cloud is updated to  a new layout recently.
But in our HC instance, the new layout is not update.
We have not deploy any code since the HC instance is setup.

May I know the schedule when will the new layout deploy to custom HC?

Beside, I found the "the ability to customize magic link email text" is also not deployed to custom HC instance.

**Comment by robinkwilson** [comment, marked as answer]
The latest Hubs Cloud release released last week! You should be able to customize the magic link email text.

The UI update will be pushed to Hubs Cloud instances after March 25th. You can learn more here: https://hubs.mozilla.com/docs/hubs-cloud-customizing-look-and-feel.html#an-announcement-about-the-hubs-user-interface

^^ The above documentation should also be linked in the admin panel.

**Reply by safeisnotanoption** [reply]
Hello. It's already mid-April. Could you please tell me when this UI update for Hubs Cloud instances will take place?

**Reply by lihong-xmuseum** [reply]
Hello, I want to know when Hubs Cloud will update the new UI. I have been waiting for a long tim
